roles/pulibrary.sneakers_worker/tasks/main.yml

```diff
@@ -1,11 +1,23 @@
 ---
 - name: Install startup script for sneakers workers
-  template: "src=sneakers.service dest=/etc/systemd/system/{{sneakers_worker_name}}.service"
+  template:
+    src: sneakers.service
+    dest: "/etc/systemd/system/{{ sneakers_worker_name }}.service"
   notify: 'restart sneakers worker'
+
 - name: Keep workers running
-  service: "name={{sneakers_worker_name}} enabled=yes state=started"
+  service:
+    name: "{{ sneakers_worker_name }}"
+    enabled: true
+    state: started
```

Comment:
Can the service state in the container also be tested given this?  If not, I don't think that this should stop anyone from merging this PR.

Reply:
our container doesn't have systemd so it cannot.